docusaurus.config.js

```diff
@@ -81,6 +81,7 @@ const config = {
           "gbz80.7",
           "rgbasm.1",
           "rgbasm.5",
+          "rgbasm-old.5",
```

Comment:
```suggestion
```

Remove the redirection until 0.9.0 is released and becomes the new `latestStable`.